docs/source/get_started/install.md

```diff
@@ -134,7 +134,7 @@ You should see an ASCII art graphic and the Kedro version number. For example:
 
 ![](../meta/images/kedro_graphic.png)
 
-If you do not see the graphic displayed, or have any issues with your installation, check out the [searchable archive of Slack discussions](https://www.linen.dev/s/kedro), or post a new query on the [Slack organisation](https://slack.kedro.org).
+If you do not see the graphic displayed, or have any issues with your installation, check out the [searchable archive of Slack discussions](https://linen-slack.kedro.org/), or post a new query on the [Slack organisation](https://slack.kedro.org).
```

Comment:
A question about writing style: do we prefer `do not`, `don't`, or have no preference? I see both used throughout our docs.

Reply:
Good question, no real answer. I'm happy with either and don't have sufficient bandwidth to enforce one or other, even if I had a preference. There's a ticket in this sprint for setting up a language "linter" which we could use to offer consistency one way or the other. But I think life's too short and each communicates perfectly clearly. So I doubt we'll enable that rule.